test/tests/selftest/pbs_syslog_tests.py

```diff
@@ -0,0 +1,146 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.selftest import *
+from ptl.utils.pbs_logutils import PBSLogUtils
+
+
+class TestSyslog(TestSelf):
+    """
+    Test pbs syslog logging
+    """
+    du = DshUtils()
+    lu = PBSLogUtils()
+
+    def setup_syslog(self, hostname=None, local_log=1,
+                     syslog_facility=1, syslog_severity=9):
+        """
+        Setup syslog in pbs.conf
+        """
+        if hostname is None:
+            hostname = self.server.hostname
+        a = {'PBS_SYSLOG': syslog_facility,
+             'PBS_SYSLOGSEVR': syslog_severity,
+             'PBS_LOCALLOG': local_log}
+        self.du.set_pbs_config(hostname=hostname, confs=a, append=True)
+        PBSInitServices().restart()
+        self.assertTrue(self.server.isUp(), 'Failed to restart PBS Daemons')
+
+    def test_basic_syslog_match(self):
+        """
+        Basic test to check if logging via syslog works
+        Each individaul daemon is logged seperately
+        """
+        self.setup_syslog(local_log=0, syslog_facility=9, syslog_severity=7)
```

Comment:
I find it difficult to find out what each facility and severity number represents. Is it possible to create a lookup dict in the test or in the framework so that future tests can also use them and it is easy to understand too?
Like 
severity = {
            "emerg" : 0,
            "alert" : 1,
            "crit" : 2,
            "err" : 3,
            "warning" : 4,
            "notice" : 5,
            "info" : 6,
            "debug" : 7,
        }
then you can use it like syslog_severity=severtiy["debug"]